2023-04-29 23:03:33,225 - _internal.py:_log[line:113] - INFO:  * Restarting with windowsapi reloader
2023-04-29 23:03:34,351 - _internal.py:_log[line:113] - WARNING:  * Debugger is active!
2023-04-29 23:03:34,354 - _internal.py:_log[line:113] - INFO:  * Debugger PIN: 504-467-380
2023-04-29 23:03:34,370 - _internal.py:_log[line:113] - INFO:  * Running on http://localhost:1234/ (Press CTRL+C to quit)
2023-04-29 23:03:54,812 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:03:54] "[32mGET / HTTP/1.1[0m" 302 -
2023-04-29 23:03:54,828 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\jinja2\\__pycache__\\ext.cpython-39.pyc', reloading
2023-04-29 23:03:54,830 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:03:54] "[37mGET /login/index HTTP/1.1[0m" 200 -
2023-04-29 23:04:08,893 - _internal.py:_log[line:113] - INFO:  * Restarting with windowsapi reloader
2023-04-29 23:04:10,047 - _internal.py:_log[line:113] - WARNING:  * Debugger is active!
2023-04-29 23:04:10,050 - _internal.py:_log[line:113] - INFO:  * Debugger PIN: 504-467-380
2023-04-29 23:04:10,065 - _internal.py:_log[line:113] - INFO:  * Running on http://localhost:1234/ (Press CTRL+C to quit)
2023-04-29 23:04:10,221 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\__init__.cpython-39.pyc', reloading
2023-04-29 23:04:10,223 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\base.cpython-39.pyc', reloading
2023-04-29 23:04:10,225 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\json.cpython-39.pyc', reloading
2023-04-29 23:04:10,225 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\pysqlcipher.cpython-39.pyc', reloading
2023-04-29 23:04:10,226 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\pysqlite.cpython-39.pyc', reloading
2023-04-29 23:04:10,227 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\dml.cpython-39.pyc', reloading
2023-04-29 23:04:10,228 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\sqlalchemy\\dialects\\sqlite\\__pycache__\\aiosqlite.cpython-39.pyc', reloading
2023-04-29 23:04:10,236 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:04:10] "[32mGET /login/index?Username=root&Password=88888888 HTTP/1.1[0m" 302 -
2023-04-29 23:04:10,257 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\encodings\\__pycache__\\unicode_escape.cpython-39.pyc', reloading
2023-04-29 23:04:10,258 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:04:10] "[37mGET /device/index HTTP/1.1[0m" 200 -
2023-04-29 23:04:10,599 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\site-packages\\flask\\__pycache__\\debughelpers.cpython-39.pyc', reloading
2023-04-29 23:04:10,602 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:04:10] "[37mGET /device/data_operation?page=1&perPage=10 HTTP/1.1[0m" 200 -
2023-04-29 23:04:42,692 - _internal.py:_log[line:113] - INFO:  * Restarting with windowsapi reloader
2023-04-29 23:04:43,756 - _internal.py:_log[line:113] - WARNING:  * Debugger is active!
2023-04-29 23:04:43,759 - _internal.py:_log[line:113] - INFO:  * Debugger PIN: 504-467-380
2023-04-29 23:04:43,775 - _internal.py:_log[line:113] - INFO:  * Running on http://localhost:1234/ (Press CTRL+C to quit)
2023-04-29 23:04:46,300 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:04:46] "[32mGET / HTTP/1.1[0m" 302 -
2023-04-29 23:04:46,315 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:04:46] "[37mGET /login/index HTTP/1.1[0m" 200 -
2023-04-29 23:04:47,375 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:04:47] "[32mGET /login/index?Username=root&Password=88888888 HTTP/1.1[0m" 302 -
2023-04-29 23:04:47,670 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:04:47] "[37mGET /device/index HTTP/1.1[0m" 200 -
2023-04-29 23:04:48,250 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:04:48] "[37mGET /device/data_operation?page=1&perPage=10 HTTP/1.1[0m" 200 -
2023-04-29 23:04:49,464 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:04:49] "[37mGET /device/data HTTP/1.1[0m" 200 -
2023-04-29 23:04:49,769 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:04:49] "[37mGET /device/data_operation?perPage=10&page=1 HTTP/1.1[0m" 200 -
2023-04-29 23:05:15,977 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\os.py', reloading
2023-04-29 23:05:16,062 - _internal.py:_log[line:113] - INFO:  * Detected change in 'S:\\anaconda3\\Lib\\importlib\\__init__.py', reloading
2023-04-29 23:07:58,132 - _internal.py:_log[line:113] - INFO:  * Restarting with windowsapi reloader
2023-04-29 23:07:59,230 - _internal.py:_log[line:113] - WARNING:  * Debugger is active!
2023-04-29 23:07:59,233 - _internal.py:_log[line:113] - INFO:  * Debugger PIN: 504-467-380
2023-04-29 23:07:59,250 - _internal.py:_log[line:113] - INFO:  * Running on http://localhost:1234/ (Press CTRL+C to quit)
2023-04-29 23:08:01,205 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:08:01] "[32mGET / HTTP/1.1[0m" 302 -
2023-04-29 23:08:01,226 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:08:01] "[37mGET /login/index HTTP/1.1[0m" 200 -
2023-04-29 23:08:02,331 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:08:02] "[32mGET /login/index?Username=root&Password=88888888 HTTP/1.1[0m" 302 -
2023-04-29 23:08:02,634 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:08:02] "[37mGET /device/index HTTP/1.1[0m" 200 -
2023-04-29 23:08:03,224 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:08:03] "[37mGET /device/data_operation?page=1&perPage=10 HTTP/1.1[0m" 200 -
2023-04-29 23:08:05,707 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:08:05] "[37mPUT /device/data_operation/1 HTTP/1.1[0m" 200 -
2023-04-29 23:08:06,020 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:08:06] "[37mGET /device/data_operation?page=1&perPage=10 HTTP/1.1[0m" 200 -
2023-04-29 23:08:07,596 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:08:07] "[37mPUT /device/data_operation/3 HTTP/1.1[0m" 200 -
2023-04-29 23:08:07,917 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:08:07] "[37mGET /device/data_operation?page=1&perPage=10 HTTP/1.1[0m" 200 -
2023-04-29 23:08:08,634 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:08:08] "[37mPUT /device/data_operation/5 HTTP/1.1[0m" 200 -
2023-04-29 23:08:08,964 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:08:08] "[37mGET /device/data_operation?page=1&perPage=10 HTTP/1.1[0m" 200 -
2023-04-29 23:08:09,938 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:08:09] "[37mPUT /device/data_operation/9 HTTP/1.1[0m" 200 -
2023-04-29 23:08:10,271 - _internal.py:_log[line:113] - INFO: 127.0.0.1 - - [29/Apr/2023 23:08:10] "[37mGET /device/data_operation?page=1&perPage=10 HTTP/1.1[0m" 200 -
